Lab1/Lab1/Models/Subjects.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Lab1.Models
+{
+    //[DataContract]
+    public class Subjects
+    {
+        //[DataMember]
+        public const string _name = "Average subject marks:";
```

Comment:
The name of the variable should be `Name`

Reply:
You said, that constants must start with "_" symbol, at classroom. May be you means _Name?

Reply:
Nope, I said that private non-const field should have name start with "_" symbol.
All public field shouldn't have name starts with this symbol:
Public Const: NAME
Public Non-const: Name
Private Const: NAME
Private Non-const: _name

Listen me carefully and read code naming conventions.